language/stdlib/modules/AccountLimits.move

```diff
@@ -170,7 +170,7 @@ module AccountLimits {
         to_limit: signer;
         limit_address: address;
         include Roles::AbortsIfNotLibraRoot{account: lr_account};
-        include Roles::AbortsIfNotParentVaspOrChildVasp{account: to_limit}; // Only ParentVASP and ChildVASP can have the account limits [E1][E2][E3][E4][E5][E6][E7].
+        include Roles::AbortsIfNotParentVaspOrChildVasp{account: to_limit}; // Only ParentVASP and ChildVASP can have the account limits [[E1]][ROLE][[E2]][ROLE][[E3]][ROLE][[E4]][ROLE][[E5]][ROLE][[E6]][ROLE][[E7]][ROLE].
```

Comment:
This doesn't fit on a line even if rendered. Please fix.

Reply:
Done.